source/plugins/go/input/lib/cadvisor.go

```diff
@@ -1051,6 +1051,18 @@ func getContainerCpuMetricItems(metricInfo map[string]interface{}, hostName, met
 						if len(os.Getenv("AZMON_RESOURCE_OPTIMIZATION_ENABLED")) > 0 {
 							telemetryProps["resoureceOptimizationEnabled"] = os.Getenv("AZMON_RESOURCE_OPTIMIZATION_ENABLED")
 						}
+						if len(os.Getenv("AZMON_KUBERNETES_METADATA_ENABLED")) > 0 {
```

Comment:
add dashboard tile to track the adoption.

Reply:
will add that